react-components/src/collaborators/CollaboratorListIcon.js

```diff
@@ -0,0 +1,24 @@
+import messages from "./messages";
+import styles from "./styles";
+import withI18N, { getMessage } from "../util/I18NWrapper";
+
+import ListAlt from "@material-ui/icons/ListAlt";
+import React from "react";
+import ToolTip from "@material-ui/core/Tooltip";
+import { withStyles } from "@material-ui/core/styles";
```

Comment:
dangit, it lost my comment. are we planning to implement https://reactjs.org/docs/code-splitting.html#import or something at some point?

Reply:
Ha... I think I posted early on like, "We should keep this in mind!" and then I forgot about it.  I'll start looking into it - thanks for the reminder! 😄